web/elm/src/Login/Login.elm

```diff
@@ -65,16 +79,24 @@ viewLoginState userState isUserMenuExpanded =
             ]
 
         UserStateLoggedIn user ->
+            let
+                displayName =
+                    userDisplayName user
+            in
             [ Html.div
                 ([ id "login-container"
                  , onClick <| Click UserMenu
                  ]
                     ++ Styles.loginContainer
                 )
-                [ Html.div (id "user-id" :: Styles.loginItem)
-                    (Html.div
-                        Styles.loginText
-                        [ Html.text (userDisplayName user) ]
+                [ Html.div
+                    (id "user-id"
+                        :: Styles.loginItem
+                        ++ [ onMouseEnter <| Hover <| Just (UserDisplayName displayName)
+                           , onMouseLeave <| Hover Nothing
+                           ]
+                    )
+                    (Html.text displayName
```

Comment:
Because the logout button is a child of the `user-id` div, the tooltip also appears when hovering over the logout button, which feels really weird. 
```html
<div id="user-id" onMouseEnter=...>
    "user-with-a-very-long-name"
    <div id="logout-button">logout</div>
</div>
```

Can you rename the parent container and create a new child div just for the username container? Something like:
```html
<div id="login-menu">
    <div id="user-id" onMouseEnter=...>"user-with-a-very-long-name"</div>
    <div id="logout-button">logout</div>
</div>
```